core/templates/dev/head/services/explorationContextService.js

```diff
@@ -100,6 +100,21 @@ oppia.factory('explorationContextService', [
       isInExplorationEditorMode: function() {
         return (this.getPageContext() === PAGE_CONTEXT.EDITOR &&
             this.getEditorTabContext() === EDITOR_TAB_CONTEXT.EDITOR);
+      },
+
+      // Following functions help in remembering whether video is already
```

Comment:
A minor question ?
This documentation you provided should be included in that new service file, right ?
and should I also add oppia authors license comment that can be seen in other js files ?
Thanx